### PR TITLE
Validate URLs when submitted

### PR DESCRIPTION
**Pull request by cguess**
This validates all the submission URLS when they're passed in, just as a secondary check since Hypatia is designed to work separately from Zenodotus if necessary.

To test:
`rails t` should take care of it
Try sending in invalid urls and hope it fails as well.

### CODE REVIEW

File: app/media_sources/facebook_media_source.rb [hunk 1]
```diff
@@ -61,7 +61,11 @@ def initialize(url)
   # @return [Forki::Post]
   def retrieve_facebook_post
     # Unlike Zorki, Forki expects a full URL
-    Forki::Post.lookup(url)
+    # begin
+      Forki::Post.lookup(url)
+    # rescue NoMethodError => e
+    #   debugger
+    # end
```
Comment (reefdog) [inline app/media_sources/facebook_media_source.rb:68]: We don't want the rescue?

Comment (reefdog) [reply app/media_sources/facebook_media_source.rb:68]: Ah, lol, it's just for the debugger.

**Review by reefdog — COMMENTED** [review]
So I'm a little confused by this, it doesn't look like there are actually any changes that achieve what it says on the tin (validating URLs). Unless that all happens upstream in the zorki and forki gems, but those are pinned to your local filesystem so I'd not approve that for merging to `master`. There does seem to be an improvement to the Instagram URL validation regex, and a cool macOS notification feature.

Anyway, maybe we should talk this one through so I know what I'm testing / what it's supposed to do!

**Comment by cguess** [comment]
OK, they should not be pegged, obviously. I'm going to looking wtf is happening here

**Pull request merged**